plugin/src/main/java/com/exadel/aem/toolkit/plugin/metadata/InterfaceHandler.java

```diff
@@ -611,6 +863,11 @@ public String toString() {
         return StringUtils.stripEnd(result.toString(), DialogConstants.SEPARATOR_SEMICOLON) + DialogConstants.CLOSING_CURLY;
     }
 
+    /**
+     * Called by {@link InterfaceHandler#toString()} to convert the given array to a string representation
+     * @param array The array to convert
+     * @return A string representation of the array; can be an empty string but never {@code null}
```

Comment:
>can be an empty string
I don't see how this can be true with the current implementation of this method....